### ISSUE TITLE
"Create variations from all attributes" seems to not work for 2 attributes like size and color

### ISSUE THREAD

**Issue by greguly** [woothemes/woocommerce#9621]
Hi,

With these settings: 
![attributes](https://cloud.githubusercontent.com/assets/303022/11191667/942bfc24-8c83-11e5-9af0-127d774d7adc.png)

and starting from 
![before](https://cloud.githubusercontent.com/assets/303022/11191687/a84584d2-8c83-11e5-8920-904b3be208f5.png)

one gets

![after](https://cloud.githubusercontent.com/assets/303022/11191695/b419591e-8c83-11e5-95f9-2bcdd9bc94e1.png)

instead of expected

![expected](https://cloud.githubusercontent.com/assets/303022/11191738/000c4e9e-8c84-11e5-86f4-20a13f9e9367.png)

In summary, instead of generating all sizes and all colors, it generates all sizes with 'Any Color...'.

I can provide more details if needed.

Cheers,
Gabriel

**Comment by greguly** [comment]
Thanks for testing :-)

It must be a conflict with some other plugin.
